utils/myspell.py

```diff
@@ -1,27 +1,25 @@
 #! /usr/bin/env python
 
-import os
 from enchant.checker import SpellChecker
-from enchant.tokenize import Filter, EmailFilter, URLFilter
+from enchant.tokenize import EmailFilter, URLFilter
 from enchant import DictWithPWL
 
 
 def myspell(fname):
     my_dict = DictWithPWL('en_US', 'mywords.txt')
     print(my_dict)
 
-    spell_checker = SpellChecker(my_dict, filters = [EmailFilter, URLFilter])
+    spell_checker = SpellChecker(my_dict, filters=[EmailFilter, URLFilter])
 
     fp = open(fname, 'r')
 
     lc = 1
     for x in fp:
-      spell_checker.set_text(x)
-      for error in spell_checker:
-        print("Error:", error.word, "(line", lc, ")")
-      lc = lc+1
+        spell_checker.set_text(x)
+        for error in spell_checker:
+            print("Error:", error.word, "(line", lc, ")")
```

Comment:
Should this print to stderr?
print("Error:", error.word, "(line", lc, ")", file=sys.stderr)

Reply:
done